### PR TITLE
[DOCS] Add a Spark DataFrame example

### PR DESCRIPTION
**Pull request by chengyineng38**
Changes proposed in this pull request:
- Add a Spark DataFrame example. This brings visibility to the SparkDFDataset API.

### CODE REVIEW

**Comment by eugmandel** [comment]
@chengyineng38 Thank you for submitting! Will review in the next day or so.

File: docs/guides/tutorials/explore_expectations_in_a_notebook.rst [hunk 1]
```diff
@@ -60,6 +62,18 @@ All of these steps take place within your notebook:
...
+    
+    - a ``Spark DataFrame``, you can use ``SparkDFDataset``:
+    
+    .. code-block:: python
+    
+        from great_expectations.dataset.sparkdf_dataset import SparkDFDataset
+    
+        my_df = SparkDFDataset(my_spark_dataframe)
+    
+    This method will convert your ``Spark DataFrame`` into a class with access to Great Expectations' methods, such as ``ProfilingResultsPageRenderer``.
```
Comment (eugmandel) [inline docs/guides/tutorials/explore_expectations_in_a_notebook.rst]:
```suggestion
    This method will create an object with access to Great Expectations' methods, such as ``ProfilingResultsPageRenderer``.
```

Comment (eugmandel) [reply docs/guides/tutorials/explore_expectations_in_a_notebook.rst]: The "convert" language is correct for Pandas, but not precise for Spark DataFrames.

**Review by eugmandel — DISMISSED** [review]
Great edit, @chengyineng38 ! Please accept my suggestion edit, and we will get the PR in.

**Comment by chengyineng38** [comment]
Great suggestion. Just accepted the edit suggestion.

**Comment by chengyineng38** [comment]
> Great edit, @chengyineng38 ! Please accept my suggestion edit, and we will get the PR in.

@eugmandel hmm not sure why your suggestion edit resulted in assertion error?

**Comment by eugmandel** [comment]
> > Great edit, @chengyineng38 ! Please accept my suggestion edit, and we will get the PR in.
> 
> @eugmandel hmm not sure why your suggestion edit resulted in assertion error?

@chengyineng38 This looks like an intermittent failure of the Azure pipeline CI - no action is required on your part.

**Review by spbail — APPROVED** [review]
lgtm

**Pull request merged**